### PR TITLE
print shape of the tensor

### LINKED ISSUE #63855: Include tensor shape in its default print formatting
Very often one would print a tensor during debugging instead of checking its shape. It would be useful to have shape information in default tensor print formatting, e.g. at the end after grad_fn or just before grad_fn (so that shape is on the screen if a huge tensor was printed)

### PR DESCRIPTION
**Pull request by mainakpal08**
Fixes #63855

Screenshot from 2021-08-28 11-24-53

### CODE REVIEW

**Comment by mpalrocks** [comment]
@heitorschueroff can you please check this once?

File: torch/_tensor_str.py [hunk 1]
```diff
@@ -284,6 +288,10 @@ def _str_intern(inp):
     # within this function.
     # TODO(albanD) This needs to be updated when more than one level is supported
     self, tangent = torch.autograd.forward_ad.unpack_dual(inp)
+    
+    # print shape of the tensor
+    if PRINT_OPTS.show_shape:
+        suffixes.append('shape= ' + str(tuple(self.shape)))
```
Comment (heitorschueroff) [inline torch/_tensor_str.py:294]: remove space in 'shape= '

**Comment by heitorschueroff** [comment]
@mpalrocks Could you please rebase this so we can get clean CI signals

File: torch/_tensor_str.py (hunk 1 above)
Comment (mpalrocks) [reply torch/_tensor_str.py]: pushed this [here](https://github.com/pytorch/pytorch/pull/64136/commits/65ab1ff26cb9b488498b5d19434c2d03f6f419fa)

**Comment by mpalrocks** [comment]
@heitorschueroff should I create a fresh PR into master branch cherry picking the necessary commits?

**Comment by ezyang** [comment]
You need to cherry-pick the change onto the 1.9 release branch and then open a PR on that; if you just try to merge from master everything else that happened in master on the meantime will get pulled in as seen here. Closing this for now.

**Pull request closed without merging**